src/decorators/class_decorator.js

```diff
@@ -16,7 +16,7 @@ import { ALL_KEYS } from '../lib/keys';
  * @param {array} [keys] The key(s) bound to the class
  * @return {object} The higher-order function that wraps the decorated class
  */
-function componentWrapper( WrappedComponent, keys = ALL_KEYS ) {
+function componentWrapper( WrappedComponent, keys = ALL_KEYS, options={global: false} ) {
```

Comment:
Minor: All calls to decorating class will create a new object. Perhaps we should have a `defaultOptions` above and just use that if not supplied.

(Is find if you don't do it now, I'll add when I work on debounce)